build.sbt

```diff
@@ -146,7 +140,13 @@ def runtimeLibCommon: Seq[Setting[_]] = common ++ runtimeScalaSettings ++ Seq(
     "-unchecked",
     "-Xlog-reflective-calls",
     "-deprecation"
-  ),
+  )
+)
+
+def runtimeLibCommon: Seq[Setting[_]] = common ++ runtimeScalaSettings ++ Seq(
+  Dependencies.validateDependenciesSetting,
+  Dependencies.pruneWhitelistSetting,
+  Dependencies.dependencyWhitelistSetting,
```

Comment:
The diff is rendering a little strangely, but the actual change here was to move `scalacOptions` from `runtimeLibCommon` to `runtimeScalaSettings`, where they will be picked up by the `cassandra-server` and `kafka-server` projects. I have no idea why those were originally not using these options... most likely an oversight.